mod_ci/controllers.py

```diff
@@ -442,6 +442,50 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
             platform=platform, id=test.id))
 
 
+def ci_badge_windows(status, platform, test_type, branch="master"):
+    platform = test.platform
+    if platform is TestPlatform.linux:
+        if test_type == TestType.commit:
+            if state == Status.SUCCESS:
+                with open('../static/badges/Windows_success.svg',
+                          'r') as svg:
+                    print svg.read()
+            elif state == Status.FAILURE:
+                with open('../static/badges/Windows_failed.svg',
+                          'r') as svg:
+                    print svg.read()
+            else:
+                with open('../static/badges/Windows_error.svg',
+                          'r') as svg:
+                    print svg.read()
+        else:
+            return
+    else:
+        return
+
+
+def ci_badge_linux(status, platform, test_type, branch="master"):
+    platform = test.platform
```

Comment:
Same remarks apply here.

Besides, why duplicate this code just for a different badge name?